make-it/Makefile

```diff
@@ -176,8 +176,7 @@ lARMv5:
 
 #========= Use makefile designed for emscripten builds
 vjs: $(OUTPUT_DIR)
-	# Just buid everything
-	make -f EmMakefile vjs
+	emmake make -f EmMakefile vjs
```

Comment:
The emmake is needed to work on Windows and it does not seem to effect the Linux builds.
I do not know exactly why it doesn't work on Windows without the emmake command...

I saw some posts on the emscripten forums about Mac and python2 name errors that were apparently fixed in newer versions of emsdk.

Can you try
emsdk update
emsdk install sdk-1.37.9-64bit

And see if you still have errors. If so I can move the emmake version to AppVeyor.yml or add a separate make target for windows